Makefile

```diff
@@ -127,7 +127,7 @@ format:
 
 .PHONY: test
 test:
-	@$(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/test.sh ./cmd/... ./pkg/...
+	@SKIP_FETCH_TOOLS=1 $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/test.sh ./cmd/... ./pkg/...
```

Comment:
Can you add it also to `test-cov`?

Reply:
Done. Here - https://github.com/gardener/gardener-extension-provider-gcp/pull/183/commits/082dd6801beb95952bd04c5912f3d3d01497398e#diff-b67911656ef5d18c4ae36cb6741b7965R134